packages/components/src/icon-button/index.js

```diff
@@ -56,7 +56,7 @@ function IconButton( props, ref ) {
 			className={ classes }
 			ref={ ref }
 		>
-			<Icon icon={ icon } ariaPressed={ ariaPressed } />
+			<Icon icon={ icon } aria-pressed={ ariaPressed } />
```

Comment:
I'm assuming for mobile, they need the first syntax. We might want to keep the prop as is and do a special treatment in the `Icon` component for this prop like we do for the `Dashicon` component.

cc @hypest

Reply:
Shouldn't it still be `hyphen-cased` if it's a React app? Am I missing something? I'm referring to React docs here. https://reactjs.org/docs/accessibility.html#wai-aria

Reply:
We need to go through the history of commits and double-check why it started to warn. It probably is no longer converted back to `aria-pressed` in the `Icon` component or somewhere deeper. In general, `aria-pressed` has no meaning in React Native, so we should be more cautious about what we pass as props.

Reply:
Based on this comment, https://github.com/WordPress/gutenberg/pull/17228/files#r329678304

I think #17356 will fix this issue. Ref: https://github.com/WordPress/gutenberg/pull/17356/files#r328582986

Reply:
I can close this in favor of #17356